src/payloads.js

```diff
@@ -92,10 +92,11 @@ module.exports = {
                     '- Check any alerts in `chime_ops`, `rx_hut_alerts`, `gpu_hut_alerts`\n' +
                     '- Check `gpu_hut_ops` to see if Coco is reporting any issues\n' +
                     '- Check the calibration broker page in Grafana and make note of anything odd\n' +
-                    '- Check the counter here (https://theremin.chimenet.ca/graphs/jumps) for feeds with ' +
-                    'a large number of jumps (> 3-5). Check those feeds here ' +
-                    '(https://grafana.chimenet.ca/d/v_R89dSiz/correlator-input-info) to make sure they are being ' +
-                    'consistently flagged',
+                    '- Check the table under the Counter header (https://theremin.chimenet.ca/graphs/jumps) for ' +
```

Comment:
Is it possible to make this a link via Mrkdwn, e.g.
```suggestion
                    '- Check the table under the [Counter header in theremin](https://theremin.chimenet.ca/graphs/jumps) for ' +
```